Userland/Applications/Spreadsheet/main.cpp

```diff
@@ -22,18 +22,13 @@
 
 int main(int argc, char* argv[])
 {
-    if (pledge("stdio recvfd sendfd rpath unix cpath wpath thread", nullptr) < 0) {
+    if (pledge("stdio recvfd sendfd rpath fattr unix cpath wpath thread", nullptr) < 0) {
         perror("pledge");
         return 1;
     }
 
     auto app = GUI::Application::construct(argc, argv);
 
-    if (pledge("stdio recvfd sendfd thread rpath cpath wpath unix", nullptr) < 0) {
-        perror("pledge");
-        return 1;
-    }
-
```

Comment:
this is not "re-pledging" the pledges, as you can see there are less pledges in this call than in the last one, pledge allows repeating calls to gradually continue tightening down restrictions by removing pledges that are not needed anymore.

Reply:
It has exactly the same pledges, just in a different order.
It has less pledges now, since i add fattr.